src/app/content/highlights/components/cardUtils.ts

```diff
@@ -9,13 +9,26 @@ import { assertDefined } from '../../../utils/assertions';
 import { cardMarginBottom } from '../constants';
 import { HighlightData } from '../types';
 
+export const hiddenHighlightOffset = -9999;
+
 export const getHighlightOffset = (container: HTMLElement | undefined, highlight: Highlight) => {
   if (!container || !highlight.range || !highlight.range.getBoundingClientRect) {
     return;
   }
 
   const {top, bottom } = highlight.range.getBoundingClientRect();
 
+  const highlightElement = highlight.elements[0] as HTMLElement;
+  const expandableAncestor = highlightElement
+    && highlightElement.closest('[data-type="solution"][aria-expanded="false"]');
```

Comment:
this seems super brittle, wouldn't it be just as good to check to see if the current highlight was hidden or not?